tests/exceptions/exceptions.h

```diff
@@ -16,22 +16,60 @@
  */
 namespace {
 
-inline const std::vector<sycl::errc> all_err_codes{
-    sycl::errc::success,
-    sycl::errc::runtime,
-    sycl::errc::kernel,
-    sycl::errc::accessor,
-    sycl::errc::nd_range,
-    sycl::errc::event,
-    sycl::errc::kernel_argument,
-    sycl::errc::build,
-    sycl::errc::invalid,
-    sycl::errc::memory_allocation,
-    sycl::errc::platform,
-    sycl::errc::profiling,
-    sycl::errc::feature_not_supported,
-    sycl::errc::kernel_not_supported,
-    sycl::errc::backend_mismatch};
+const std::vector<sycl::errc>& get_err_codes() {
+  static const std::vector<sycl::errc> all_err_codes{
```

Comment:
I suppose using dynamic size array is not necessary. `std::array` seems to fit better here.

Reply:
Replaced std::vector with std::array